### DISCUSSION TITLE
Hello I have encounted a issue when I run the Tutorils like blow

### DISCUSSION THREAD

**Discussion by hit-JB** [USC-ACTLab/crazyswarm#568] [Q&A]
`[ INFO] [1639814884.334823238]: CF run: 0.553540 s
update commander/enHighLevel to 1
update kalman/resetEstimation to 1
update locSrv/extPosStdDev to 0.001
update locSrv/extQuatStdDev to 0.05
update ring/effect to 16
update ring/headlightEnable to 0
update ring/solidBlue to 255
update ring/solidGreen to 0
update ring/solidRed to 0
update stabilizer/controller to 2
update stabilizer/estimator to 2
update ctrlMel/i_range_m_z to 1500
update ctrlMel/i_range_xy to 2
update ctrlMel/i_range_z to 0.4
update ctrlMel/kR_xy to 70000
update ctrlMel/kR_z to 60000
update ctrlMel/kd_omega_rp to 200
update ctrlMel/kd_xy to 0.2
update ctrlMel/kd_z to 0.4
update ctrlMel/ki_m_z to 500
update ctrlMel/ki_xy to 0.05
update ctrlMel/ki_z to 0.05
update ctrlMel/kp_xy to 0.4
update ctrlMel/kp_z to 1.25
update ctrlMel/kw_xy to 20000
update ctrlMel/kw_z to 12000
update ctrlMel/mass to 0.032
update ctrlMel/massThrust to 132000
[ INFO] [1639814884.351012401]: [cf1] Update parameters
[ INFO] [1639814884.385692982]: [cf1] CONTROLLER: Using Mellinger (2) controller
[ INFO] [1639814884.392913680]: [cf1] ESTIMATOR: Using Kalman (2) estimator
[ INFO] [1639814884.406550883]: [cf1] ESTKALMAN: State out of bounds, resetting
[ INFO] [1639814884.432281767]: Update params: 0.097386 s
[crazyswarm_server-2] process has died [pid 10599, exit code -11, cmd /home/jb/crazyflie/crazyswarm/ros_ws/devel/lib/crazyswarm/crazyswarm_server __name:=crazyswarm_server __log:=/home/jb/.ros/log/9f4975c0-5fd9-11ec-8169-54ee75af8168/crazyswarm_server-2.log].
log file: /home/jb/.ros/log/9f4975c0-5fd9-11ec-8169-54ee75af8168/crazyswarm_server-2*.log
^C[rviz-5] killing on exit
[crazyswarm_teleop-4] killing on exit
[joy-3] killing on exit
[rosout-1] killing on exit
[master] killing on exit
shutting down processing monitor...
... shutting down processing monitor complete
`

**Comment by hit-JB** [comment]
When i run the crazyflie_sever.launch the process died after the process encounted the following message
`[ INFO] [1639816021.437419712]: [cf1] Update parameters
[ INFO] [1639816021.476421941]: [cf1] CONTROLLER: Using Mellinger (2) controller
[ INFO] [1639816021.480181805]: [cf1] ESTIMATOR: Using Kalman (2) estimator
[ INFO] [1639816021.491970662]: [cf1] ESTKALMAN: State out of bounds, resetting
[ INFO] [1639816021.500258342]: Update params: 0.117833 s
[crazyswarm_server-2] process has died [pid 11715, exit code -11, cmd /home/jb/crazyflie/crazyswarm/ros_ws/devel/lib/crazyswarm/crazyswarm_server __name:=crazyswarm_server __log:=/home/jb/.ros/log/439943ec-5fdc-11ec-8169-54ee75af8168/crazyswarm_server-2.log].
log file: /home/jb/.ros/log/439943ec-5fdc-11ec-8169-54ee75af8168/crazyswarm_server-2*.log
`

**Comment by hit-JB** [comment]
Could you please give me some advice

**Reply by whoenig** [reply]
The error you posted indicates a segfault of the server. Most of the time this is caused by incorrect configuration files. You can post your configuration (launch file and any other files you edited), or attach a debugger, see http://wiki.ros.org/roslaunch/Tutorials/Roslaunch%20Nodes%20in%20Valgrind%20or%20GDB.

**Reply by hit-JB** [reply]
ok thaks
